submissions/maryOkosun/phpDatabase/viewQuery.php

```diff
@@ -0,0 +1,40 @@
+<?php
+include("cookie.php");
+$results = isset($_COOKIE['view'])? unserialize($_COOKIE['view']) : $View->viewcustomersDetails($id);
```

Comment:
Where is this variable `$View` coming from?

Reply:
@adaugochi 
The `$View` is gotten from class name in cookie2.php file

Reply:
I can only see `class View`